var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -41,6 +44,10 @@ class NeurodamusModel(SimModel):
     def build_model(self, spec, prefix):
         """Build and install the bare model.
         """
+        if spec.satisfies('@develop'):
+            real_version = which('git')('describe', '--tags', output=str)
+            tty.warn('Building develop version. Real version is ' + real_version)
```

Comment:
this is more for developer / debugging. Is that necessary?

Reply:
that is more for developer --> yes, especially myself to avoid I blindly build the wrong develop

Reply:
If we are just using `tty.warn`, I would be in favour of removing it